web.browser/src/actions/fetchActions.js

```diff
@@ -0,0 +1,40 @@
+export const LOADPOSTS = 'LOADPOSTS';
+const LOADCATEGORIES = 'LOADCATEGORIES';
+export const LOADWEEKS = 'LOADWEEKS';
+
+export function loadPosts(items) {
+  return {
+    type: LOADPOSTS,
+    payload: items,
+  };
+}
+export function loadCategories(items) {
+  return {
+    type: LOADCATEGORIES,
+    payload: items,
+  };
+}
+export function loadWeeks(items) {
+  return {
+    type: LOADWEEKS,
+    payload: items,
+  };
+}
+export const fetchPosts = nothing => (dispatch) => {
+  fetch('http://localhost:8000/posts', {})
```

Comment:
You're re-using `'http://localhost:8000/` in three methods. Store in a var and use template strings.